app/views/admin/attachments/_form.html.erb

```diff
@@ -32,12 +48,28 @@
       <% end %>
     <% end %>
   <% elsif attachment.is_a?(ExternalAttachment) %>
-    <%= form.text_field :external_url %>
+    <div class="govuk-!-margin-bottom-8">
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "External url (required)"
+        },
+        name: "attachment[external_url]",
+        heading_level: 2,
+        heading_size: "l",
+        value: form.object.external_url.blank? ? "" : form.object.external_url
+      } %>
+    </div>
   <% else %>
     <%= render 'attachment_data_fields', form: form %>
   <% end %>
 
   <%= hidden_field_tag :type, params[:type] %>
 
-  <%= form.save_or_cancel cancel: attachable_attachments_path(attachable) %>
+  <% if attachment.is_a?(ExternalAttachment) %>
+    <%= render "govuk_publishing_components/components/button", {
+      text: "Next"
+    } %>
+  <% else %>
+    <%= form.save_or_cancel cancel: attachable_attachments_path(attachable) %>
```

Comment:
We should probably replace this with govuk buttons

Reply:
I've just kept the existing buttons here for now as it's out of scope for external attachments. This will be replaced when the other forms of attachments are being worked on.

Reply:
I think we should add in the styling here because everything else is using govuk design system except this:

```erb
<div class="govuk-button-group">
  <%= render "govuk_publishing_components/components/button", {
    text: "Save"
  } %>

  <a class="govuk-link" href="<%= attachable_attachments_path(attachable) %>">Cancel</a>
</div>
```